<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>845</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="home">
       <attribute name="title">
        <string>Главная</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="loadTable">
          <property name="text">
           <string>Загрузить таблицу</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="messagesHome">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1034</width>
             <height>705</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="database">
       <attribute name="title">
        <string>database.sqlite</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0,0">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="nameLabel">
             <property name="text">
              <string>Имя</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="searchPatientTextField">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>Введите имя пациента</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="fnameLabel">
             <property name="text">
              <string>Фамилия</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="fnamTextField">
             <property name="placeholderText">
              <string>Введите фамилию пациента</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="surnameLabel">
             <property name="text">
              <string>Отчество</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="surnameTextField">
             <property name="placeholderText">
              <string>Введите отчество пациента</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="searchPatientButton">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Найти пациента</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="findPatients"/>
        </item>
        <item>
         <widget class="QPushButton" name="generatePDF">
          <property name="text">
           <string>Сгенерировать PDF</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="preview_button">
          <property name="text">
           <string>Предпросмотр</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1034</width>
             <height>611</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTableWidget" name="tableWidget">
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>300</number>
              </attribute>
              <row/>
              <column>
               <property name="text">
                <string>Рекомендуемый срок</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Инфекционное заболевание</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="addLineButton">
              <property name="text">
               <string>Добавить строку</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteLineButton">
              <property name="text">
               <string>Удалить строку</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1078</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
